bemani/utils/read.py

```diff
@@ -1811,6 +1785,79 @@ def __revivals(self, songid: int, chart: int) -> Optional[int]:
             if old_id is not None:
                 return old_id
 
+legg_map = {
+    1017: 1100, 4005: 4100,
```

Comment:
The double-things-on-line threw me, but I don't think its worth fixing. Honestly we really should just say "We are formatted with black", do a bite-the-bullet pass and move on from there.